src/cascadia/TerminalControl/TerminalControlLib.vcxproj

```diff
@@ -110,61 +126,22 @@
     </ClCompile>
+    <Link>
+      <AdditionalDependencies>dwrite.lib;dxgi.lib;d2d1.lib;d3d11.lib;shcore.lib;winmm.lib;pathcch.lib;propsys.lib;uiautomationcore.lib;Shlwapi.lib;ntdll.lib;user32.lib;kernel32.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <!-- <AdditionalDependencies>WindowsApp.lib;user32.lib;shell32.lib;%(AdditionalDependencies)</AdditionalDependencies> -->
+    </Link>
+    <Reference>
+      <Private>false</Private>
+    </Reference>
   </ItemDefinitionGroup>
-  <Import Project="$(SolutionDir)build\rules\CollectWildcardResources.targets" />
+  <!-- ========================= Globals ======================== -->
+  <Import Project="$(OpenConsoleDir)src\cppwinrt.build.post.props" />
 
-  <!--
-    We're linking a static library that has the same root namespace as us. Unfortunately,
-    if we allow a winmd that contains our namespace into the to-be-XamlCompiled reference list
-    the xaml compiler will become upset with us and fail to find any types defined in _this_
-    version of the namespace. Therefore, we have to strip them out.
-    Because ReferencePath is an important item group in MSBuild, we're going to make sure to
-    restore it the moment Xaml is done with it.
-    We have to use a static library with the _same_ namespace as ours because there's a weird
-    invariant somewhere in WinRT that every library project produce a single namespace. We're
-    going to let TerminalCore (static lib) contribute to our namespace, but when we do that
-    we get the xaml compiler complaining.
-  -->
-  <PropertyGroup>
-    <MarkupCompilePass1DependsOn>$(MarkupCompilePass1DependsOn);OpenConsoleStripDuplicateWinmdFromReferencesBeforePass1</MarkupCompilePass1DependsOn>
-    <MarkupCompilePass2DependsOn>$(MarkupCompilePass2DependsOn);OpenConsoleStripDuplicateWinmdFromReferencesBeforePass2</MarkupCompilePass2DependsOn>
-  </PropertyGroup>
-  <Target Name="OpenConsoleStripDuplicateWinmdFromReferencesBeforePass1">
-    <ItemGroup>
-      <!-- Back up the references we're about to delete -->
-      <OCReferencePathToRestore Include="@(ReferencePath)" Condition="'%(Filename)' == '$(RootNamespace)'"/>
-      <ReferencePath Remove="@(OCReferencePathToRestore)"/>
-    </ItemGroup>
-  </Target>
-  <Target Name="OpenConsoleRestoreDuplicateWinmdToReferences" AfterTargets="MarkupCompilePass2">
-    <ItemGroup>
-      <!-- Put them back. -->
-      <ReferencePath Include="@(OCReferencePathToRestore)"/>
-    </ItemGroup>
-  </Target>
-  <Target Name="OpenConsoleStripDuplicateWinmdFromReferencesBeforePass2">
-    <ItemGroup>
-      <!-- Nobody else uses this item group, it's safe to never restore. -->
-      <WinMDReferenceToCompile Remove="@(WinMDReferenceToCompile)" Condition="'%(Filename)' == '$(RootNamespace)'"/>
-    </ItemGroup>
-  </Target>
+  <Import Project="$(SolutionDir)build\rules\CollectWildcardResources.targets" />

```

Comment:
shocked that resources might just "work", scared too

Reply:
me too